plugins/woocommerce/tests/e2e-pw/global-setup.js

```diff
@@ -60,6 +60,8 @@ module.exports = async ( config ) => {
 			await adminPage.fill( 'input[name="pwd"]', admin.password );
 			await adminPage.click( 'text=Log In' );
 			await adminPage.waitForLoadState( 'networkidle' );
+			await adminPage.goto( `/wp-admin` );
```

Comment:
Makes sense to me.  Better to not leave things up to chance, so this is a great addition.

Reply:
Yup, I just found out where the redirect came from, I've shared some details on a comment I've posted 1 min ago, not sure if you had time to see it yet. We could either go to wp-admin, or we could disable the onboarding wizard.